server/views/workflows.py

```diff
@@ -99,11 +54,12 @@ def make_init_state(request, workflow=None, modules=None):
         ret['loggedInUser'] = UserSerializer(request.user).data
 
     if workflow and not workflow.request_read_only(request):
-        ret['editCellsModuleId'] = edit_cells_module_id()
-        ret['sortModuleId'] = sort_module_id()
-        ret['reorderModuleId'] = reorder_module_id()
-        ret['renameModuleId'] = rename_module_id()
-        ret['duplicateModuleId'] = duplicate_module_id()
+        ret['updateTableModuleIds'] = {}
+        for id_name, id_alias in module_id_mapping.items():
+            # Numerous dependencies, keeping ModuleId keys
+            ret[id_alias] = module_id(id_name)
```

Comment:
Is this line still needed? It looks like you are writing the module ID to the output twice